desktop-exporter/server.go

```diff
@@ -0,0 +1,70 @@
+package desktopexporter
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+
+	"github.com/gorilla/mux"
+)
+
+type Server struct {
+	server     http.Server
+	traceStore *TraceStore
+}
+
+func getTracesHandler(traceStore *TraceStore) func(http.ResponseWriter, *http.Request) {
+	return func(writer http.ResponseWriter, request *http.Request) {
+		traceStore.mut.Lock()
+		defer traceStore.mut.Unlock()
```

Comment:
In a future PR we should define some methods to read data and push this logic into the store itself, though this is perfectly fine for the moment